bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -64,179 +74,473 @@
-        dptTypeMap.put(DPTXlator4ByteSigned.DPT_REACTIVE_ENERGY.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteSigned.DPT_ACTIVE_ENERGY_KWH.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteSigned.DPT_APPARENT_ENERGY_KVAH.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteSigned.DPT_REACTIVE_ENERGY_KVARH.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteSigned.DPT_DELTA_TIME.getID(), DecimalType.class);
-
-        // Datapoint Types "4-Octet Float Value", Main number 14
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_ACCELERATION_ANGULAR.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_ANGLE_DEG.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_ELECTRIC_CURRENT.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_ELECTRIC_POTENTIAL.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_FREQUENCY.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_POWER.getID(), DecimalType.class);
-        dptTypeMap.put(DPTXlator4ByteFloat.DPT_PRESSURE.getID(), DecimalType.class);
-
-        // Datapoint Types "String", Main number 16
-        dptTypeMap.put(DPTXlatorString.DPT_STRING_8859_1.getID(), StringType.class);
 
-        // Datapoint Types "Scene Number", Main number 17
-        dptTypeMap.put(DPTXlatorSceneNumber.DPT_SCENE_NUMBER.getID(), DecimalType.class);
-
-        // Datapoint Types "Scene Control", Main number 18
-        dptTypeMap.put(DPTXlatorSceneControl.DPT_SCENE_CONTROL.getID(), DecimalType.class);
-
-        // Datapoint Types "DateTime", Main number 19
-        dptTypeMap.put(DPTXlatorDateTime.DPT_DATE_TIME.getID(), DateTimeType.class);
+        /**
+         * MainType: 2
+         * 2.002: Boolean Controlled, values 0 false 1 true
+         * 2.003: Enable Controlled, values 0 disable 1 enable
+         * 2.011: State Controlled, values 0 inactive 1 active
+         * 2.001: Switch Controlled, values 0 off 1 on
+         * 2.012: Invert Controlled, values 0 not inverted 1 inverted
+         * 2.010: Start Controlled, values 0 stop 1 start
+         * 2.008: Up/Down Controlled, values 0 up 1 down
+         * 2.009: Open/Close Controlled, values 0 open 1 close
+         * 2.006: Binary Controlled, values 0 low 1 high
+         * 2.007: Step Controlled, values 0 decrease 1 increase
+         * 2.004: Ramp Controlled, values 0 no ramp 1 ramp
+         * 2.005: Alarm Controlled, values 0 no alarm 1 alarm
+         */
+        dptMainTypeMap.put(2, DecimalType.class);
+        /** Exceptions Datapoint Types "B2", Main number 2 */
+        // Example: dptTypeMap.put(DPTXlator1BitControlled.DPT_SWITCH_CONTROL.getID(), DecimalType.class);
+
+        /**
+         * MainType: 3
+         * 3.007: Dimming, values decrease 7 increase 7
+         * 3.008: Blinds, values up 7 down 7
+         */
```

Comment:
Are these values right, or is this a copy/paste error?

Reply:
done.